computer-use-server/mcp_tools.py

```
@@ -1250,6 +1254,24 @@ def set_context_from_headers(headers: dict):
         current_mcp_servers.set(headers["x-openwebui-mcp-servers"])
 
+    # MCP config (full JSON, base64-encoded) — takes priority over CSV list
+    import base64 as _base64
+    mcp_config_b64 = (
+        headers.get("x-mcp-config") or headers.get("x-openwebui-mcp-config") or ""
+    )
+    if mcp_config_b64:
+        try:
+            current_mcp_servers.set(_base64.b64decode(mcp_config_b64).decode())
+        except Exception:
+            pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)